tests/test_models.py

```diff
@@ -58,3 +58,11 @@ def tearDown(self):
     def test_XXXX(self):
         """ It should always be true """
         self.assertTrue(True)
+
+    def test_get_product_list(self):
+        """It should Get a list of Pets"""
+        self._create_products(5)
+        response = self.client.get(BASE_URL)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        data = response.get_json()
+        self.assertEqual(len(data), 5)
```

Comment:
these codes should write in test_route.py because it is a test for API and you should write the test case for the Product model in test_model.py

and I think you should update your master branch otherwise it would conflict.